_includes/flipflop2.html

```diff
@@ -136,13 +140,40 @@ <h4>Press <strong>f</strong> to begin.</h4>
 				$('#stateTable').css('position', 'relative');
 				$('#stateTable').css('left', '0px');
 
-				$(document).keypress(function(event) {
+				var next = 0;
+				$(document).keydown(function(event) {
 					if (String.fromCharCode(event.which) == 'f' || String.fromCharCode(event.which) == 'F') {
 						if ($('#about').is(':visible')) {
 							$('#step1').fadeIn();
 							$('#about').hide();
+							next++;
 						}
 					}
+					if (event.which == 13) {
+						if(next === 1) {
+								$('#confirmStep1').click();
+							}
+						if (next === 2) {
+								$('#confirmStep2').click();
+							}
+							if (next === 3) {
+						 		$('#confirmSummary').click();
+						 	}
+						}
+						if (String.fromCharCode(event.which) == 'L' || String.fromCharCode(event.which) == 'l') {
+						if(next === 2) {
+								$('#backToStep1').click();
+							}
+						if (next === 3) {
+								$('#backToStep2').click();
+							}
+						}
+						if (next === 0) {
+							if (String.fromCharCode(event.which) == 'R' || String.fromCharCode(event.which) == 'r') {
```

Comment:
This condition is not optimized
why don't you use:`event.keycode===82` instead of this